splitio/recorder/recorder.py

```diff
@@ -87,6 +87,56 @@ def record_track_stats(self, event, latency):
         return self._event_sotrage.put(event)
 
 
+class StandardRecorderAsync(StatsRecorder):
+    """StandardRecorder async class."""
+
+    def __init__(self, impressions_manager, event_storage, impression_storage, telemetry_evaluation_producer):
+        """
+        Class constructor.
+
+        :param impressions_manager: impression manager instance
+        :type impressions_manager: splitio.engine.impressions.Manager
+        :param event_storage: event storage instance
+        :type event_storage: splitio.storage.EventStorage
+        :param impression_storage: impression storage instance
+        :type impression_storage: splitio.storage.ImpressionStorage
+        """
+        self._impressions_manager = impressions_manager
+        self._event_sotrage = event_storage
+        self._impression_storage = impression_storage
+        self._telemetry_evaluation_producer = telemetry_evaluation_producer
+
+    async def record_treatment_stats(self, impressions, latency, operation, method_name):
+        """
+        Record stats for treatment evaluation.
+
+        :param impressions: impressions generated for each evaluation performed
+        :type impressions: array
+        :param latency: time took for doing evaluation
+        :type latency: int
+        :param operation: operation type
+        :type operation: str
+        """
+        try:
+            if method_name is not None:
```

Comment:
do you need two params (method_name and operation), or can it be done with only one?
E.g.:
```
if operation is not None:
    await self._telemetry_evaluation_producer.record_latency(operation, latency)
```

Reply:
method_name is optional from the original code, but operation is required. So the if has to move to the previous call.
This is the original code BTW.